packages/app/components/token-picker/constants.ts

```diff
@@ -1,43 +1,16 @@
-export const TOKEN_PICKER_COMMON_TOKENS = [
-  {
-    address: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
-    name: "Wrapped XDAI",
-    symbol: "WXDAI",
-    decimals: 18,
-    logoURI: "/assets/images/tokens/wxdai.png",
-    chainId: 100
-  },
-  {
-    address: "0x6A023CCd1ff6F2045C3309768eAd9E68F978f6e1",
-    name: "Wrapped Ether on Gnosis chain",
-    symbol: "WETH",
-    decimals: 18,
-    logoURI: "/assets/images/tokens/weth.png",
-    chainId: 100
-  },
-  {
-    address: "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
-    name: "USD Coin",
-    symbol: "USDC",
-    decimals: 6,
-    logoURI: "/assets/images/tokens/usdc.png",
-    chainId: 100
-  },
-  {
-    address: "0x9C58BAcC331c9aa871AFD802DB6379a98e80CEdb",
-    name: "Gnosis Token on Gnosis chain",
-    symbol: "GNO",
-    decimals: 18,
-    logoURI: "/assets/images/tokens/gno.png",
-    chainId: 100
-  },
+import {
+  GNOSIS_GNO,
+  GNOSIS_SWAPR,
+  GNOSIS_WETH,
+  GNOSIS_WXDAI,
+  MAINNET_USDC,
+  MAINNET_WETH,
+  TokenFromTokenlist,
+} from "@/models/token";
```

Comment:
I think I would prefer to have something like:

```suggestion
import {
  gnosis,
  mainnet,
  TokenFromTokenlist,
} from "@/models/token";
```